src/events/client/messageCreate.ts

```diff
@@ -52,6 +53,14 @@ export default async function messageCreateHandler(message: Eris.Message) {
                 if (!preCheckResult) return;
             }
 
+            const gameSession = gameSessions[message.guildID];
+            if (gameSession && gameSession.gameType === GameType.COMPETITION) {
+                if (!COMPETITION_MODERATOR_IDS.includes(message.author.id) && !["skip", "score", "scoreboard", "options", "help", "news", "stats", "s", "profile", "leaderboard"].includes(parsedMessage.action)) {
```

Comment:
You can add a new prerunCheck for all of these commands instead of hardcoding list here